.cicdtemplate/.bitrise/bitrise.yml

```diff
@@ -0,0 +1,214 @@
+
+      # Execute UnitTest
+      - android-unit-test@1:
+          inputs:
+            - module: "$MODULE"
+            - variant: "$VARIANT"
+            - project_location: "$PROJECT_LOCATION"
+
+      # Execute build on $VARIANT
+      - android-build@0:
+          inputs:
+            - module: "$MODULE"
+            - variant: "$VARIANT"
+            - project_location: "$PROJECT_LOCATION"
+
+      # Upload app-debug.apk to Apps & Artifacts for later downloading
+      - deploy-to-bitrise-io@1:
+          inputs:
+            - zip_name: ''
+            - deploy_path: /bitrise/deploy/app-debug.apk
+            - notify_email_list: ''
+          is_always_run: false
+
+      - cache-push@2: {}
+
+  # Workflow on release/* branch
+  release:
+    steps:
+      - activate-ssh-key@4:
+          run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+
+      - git-clone@4: {}
+
+      - cache-pull@2: {}
+
+      - file-downloader@1:
+          inputs:
+            - destination: "$KEYSTORE_FILE"
+            - source: "$BITRISEIO_ANDROID_KEYSTORE_URL"
+
+      - install-missing-android-tools@2:
+          inputs:
+            - gradlew_path: "$PROJECT_LOCATION/gradlew"
+
+      - android-lint@0:
+          inputs:
+            - project_location: "$PROJECT_LOCATION"
+
+      - android-unit-test@1:
+          inputs:
+            - project_location: "$PROJECT_LOCATION"
+            - module: "$MODULE"
+            - variant: release
+
+      - android-build@0:
+          inputs:
+            - module: "$MODULE"
+            - project_location: "$PROJECT_LOCATION"
+            - variant: release
+
+      # Upload app-release.apk to Apps & Artifacts for downloading later
+      - deploy-to-bitrise-io@1:
+          inputs:
+            - zip_name: ''
+            - deploy_path: /bitrise/deploy/app-release.apk
+            - notify_email_list: ''
+          is_always_run: false
+
+      - cache-push@2: {}
+
+  # Workflow on master branch
+  master:
+    steps:
+
+      - activate-ssh-key@4:
+          run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+
+      - git-clone@4: {}
+
+      - cache-pull@2: {}
+
+      - install-missing-android-tools@2:
+          inputs:
+            - gradlew_path: "$PROJECT_LOCATION/gradlew"
+
+      - android-lint@0:
+          inputs:
+            - project_location: "$PROJECT_LOCATION"
+            - module: "$MODULE"
+            - variant: "$VARIANT"
+
+      - android-unit-test@1:
+          inputs:
+            - project_location: "$PROJECT_LOCATION"
+            - module: "$MODULE"
+            - variant: "$VARIANT"
+
+      # Build aab release bundle
+      - android-build@0:
+          inputs:
+            - variant: release
+            - build_type: aab
+            - module: "$MODULE"
+
+      # Upload to Google Play Store
+      - google-play-deploy@3:
+          inputs:
+            - package_name: "$PACKAGE_NAME"
+            - service_account_json_key_path: "$BITRISEIO_SERVICE_ACCOUNT_JSON_KEY_URL"
```

Comment:
I think we have to define `app_path: path to aab file`, no?

Reply:
By default, `google-play-deploy` will using 2 paths for supporting multiple artifacts (APK and AAB) then it doesn't display here. I updated to use AAB only 🙏